caddy/command.go

```diff
@@ -0,0 +1,195 @@
+package caddy
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/caddyserver/caddy/v2"
+	"github.com/caddyserver/caddy/v2/caddyconfig"
+	caddycmd "github.com/caddyserver/caddy/v2/cmd"
+	"github.com/caddyserver/caddy/v2/modules/caddyhttp"
+	"github.com/caddyserver/caddy/v2/modules/caddyhttp/fileserver"
+	"github.com/caddyserver/caddy/v2/modules/caddyhttp/rewrite"
+	"github.com/caddyserver/certmagic"
+	"go.uber.org/zap"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	caddycmd.RegisterCommand(caddycmd.Command{
+		Name:  "php-server",
+		Usage: "[--domain <example.com>] [--root <path>] [--listen <addr>] [--access-log]",
+		Short: "Spins up a production-ready PHP server",
+		Long: `
+A simple but production-ready PHP server. Useful for quick deployments,
+demos, and development.
+
+The listener's socket address can be customized with the --listen flag.
+
```

Comment:
Could maybe mention in here "This is effectively the same as running with Caddyfile that looks like this..." So that it ships with an example documented in case they need to "eject" and customize it with extra rewrites or whatever.